js/game.js

```diff
@@ -1,4 +1,6 @@
 var rats = document.querySelectorAll('.rat')
+var gameTimer = 0
+var frequency = 1500
 
 /**
```

Comment:
consider deleting doc block for something that isn't a function (change to comment)